app/menu.js

```diff
@@ -1,11 +1,15 @@
+'use strict';
```

Comment:
If anyone else is confused try this in node 5
```
var a = [1,2,3,4,5,6]; for (const b of a) { console.log(b); }
```
vs 
```
'use strict'; var a = [1,2,3,4,5,6]; for (const b of a) { console.log(b); }
```

Removing the `'use strict';` would essentially mean dropping support for anything less than node 6.